docs-kits/kits/ESS-Kit/page_adoption-view.md

```diff
@@ -174,7 +181,7 @@ The data model follows the Catena-X Standard [CX-0144](<https://catena-x.net/de/
 
 ### Access & Usage Policies / Code of Conduct
 
-#### Code of Conduct of the Catena-X Use Case Sustainability / Environmental and Social standards
+Code of Conduct of the Catena-X Use Case Sustainability / Environmental and Social standards
```

Comment:
From my perspective, the #### was correct? Isn't it a header?

Reply:
instead of just bold line, use header level 4 (####)